lib/plugins/jsonBodyParser.js

```diff
@@ -28,8 +28,13 @@ function jsonBodyParser(options) {
         // save original body on req.rawBody and req._body
         req.rawBody = req._body = req.body;
 
-        if (req.getContentType() !== 'application/json' || !req.body) {
-            return next();
+        var jsonPatternMatcher = new RegExp('^application/[a-zA-Z.]+\\+json');
```

Comment:
This Regexp is used in two different places. Perhaps make it a `const` and share?